### PR TITLE
Update RPC Spec for Media Skip Indicators

### LINKED ISSUE #127: [SDL 0198] - Media Skip Indicators
Proposal: [Media Skip Indicators](https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0198-media-skip-indicators.md)

> This feature is similar to [SDL-0109 Audio Streaming Indicator](https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0109-set-audio-streaming-indicator.md) except that it affects the next and previous buttons. These buttons should have the ability to show time skip buttons that are commonly used by podcast & audiobook media. 

Review: https://github.com/smartdevicelink/sdl_evolution/issues/596

Steering Committee Decision:

> The Steering Committee voted to accept this proposal with the following revisions:
- Incorporate the feedback agreed to [in this comment](https://github.com/smartdevicelink/sdl_evolution/issues/596#issuecomment-418749992) by the author.
 - Ensure that conditions of `SeekIndicatorType` are documented, as requested [in this comment](https://github.com/smartdevicelink/sdl_evolution/issues/596#issuecomment-418755274). 

> The proposal .md file was updated to reflect these revisions on 9/20/18.

### PR DESCRIPTION
**Pull request by RHenigan**
Fixes #127 
This PR will add the changes needed from the SDL 0198 Proposal for Media Skip Indicators to be able to be set to skip a track at a time or to skip a set amount of time

### CODE REVIEW

**Comment by RHenigan** [comment]
[(3) [SDL 0198] - Media Skip Indicators #871 [0.5]](https://trello.com/c/3Sw6FUMh/515-3-sdl-0198-media-skip-indicators-871-05)

File: MOBILE_API.xml [hunk 1]
```diff
@@ -4685,6 +4685,22 @@
...
+        <element name="TRACK"/>
+        <element name="TIME"/>
+    </enum>
+
+    <struct name="SeekStreamingIndicator" since="7.1">
+        <description>
+            The seek next / skip previous subscription buttons' content
+        </description>
+
+        <param name="type" type="SeekIndicatorType" mandatory="true" />
```
Comment (ShobhitAd) [inline MOBILE_API.xml]: Extra new line between the description and param isn't needed
```suggestion
        </description>
        <param name="type" type="SeekIndicatorType" mandatory="true" />
```

**Review by ShobhitAd — CHANGES_REQUESTED** [review]
Please merge in the latest changed from develop to change the RAPI version to "7.1.0"

File: MOBILE_API.xml [hunk 2]
```diff
@@ -5980,6 +5995,16 @@
...
             </description>
         </param>
+        <param name="forwardSeekIndicator" type="SeekStreamingIndicator" mandatory="false" since="7.1">
+            <description>
+                Used to control the forward seek button to either skip forward a set amount of time or to the next track.
+            </description>
+        </param>
+        <param name="backSeekIndicator" type="SeekStreamingIndicator" mandatory="false" since="7.1">
+            <description>
+                Used to control the forward seek button to either skip back a set amount of time or to the previous track.
```
Comment (ShobhitAd) [inline MOBILE_API.xml]:
```suggestion
                Used to control the back seek button to either skip back a set amount of time or to the previous track.
```

**Review by ShobhitAd — CHANGES_REQUESTED** [review]

**Review by ShobhitAd — APPROVED** [review]

**Comment by ShobhitAd** [comment]
@RHenigan Please resolve merge conflicts

**Pull request merged**